### PR TITLE
Fix merge

### PR DESCRIPTION
**Pull request by muselesscreator**
**TL;DR -** 
Some unintended changes went in yesterday in the rush to get all 3 tickets in by end of sprint.  This PR resolves them.

**Resolved Issues: **
* removed formatting around team usernames in oa_response.html
* removed an extraneous closing div that was introduced in oa_response.html
* don't look up team submission context in staff preview (and associated test)

**Developer Checklist**
- [ ] Reviewed the [release process](./release_process.md)
- [ ] Translations up to date
- [ ] JS minified, SASS compiled
- [ ] Test suites passing on Jenkins
- [ ] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA: https://openedx.atlassian.net/browse/EDUCATOR-5292
FIY: @edx/masters-devs-gta

### CODE REVIEW

File: openassessment/templates/openassessmentblock/response/oa_response.html [hunk 1]
```diff
@@ -139,7 +139,6 @@ <h5 class="submission__answer__part__text__title">{% trans "The prompt for this
                                   </strong>
                                 {% endif %}
                             </div>
-                            </div>
```
Comment (jansenk) [inline openassessment/templates/openassessmentblock/response/oa_response.html:142]: how

Comment (muselesscreator) [reply openassessment/templates/openassessmentblock/response/oa_response.html:142]: no idea.  I think because we touched neighboring divs.

**Comment by nsprenkle** [comment]
Please close duplicate #1477

File: openassessment/xblock/test/test_submission.py [hunk 2]
```diff
@@ -750,11 +751,28 @@ def test_get_team_submission_context(
...
+        xblock.xmodule_runtime = Mock(
+            user_is_staff=False
+        )
         xblock.get_team_info = Mock(return_value=team_info)
         context = {}
         xblock.get_team_submission_context(context)
         self.assertEqual(context, {})
 
+    @scenario('data/submission_open.xml', user_id="Red Five")
+    def test_get_team_submission_context__staff_view(self, xblock):
```
Comment (nsprenkle) [inline openassessment/xblock/test/test_submission.py:763]: This test itself is not really explanatory, outside exceptions not being thrown. Would like to see a docstring or comment explaining what you're trying to assert.
